pages/index.js

```diff
@@ -2,13 +2,26 @@ import Layout from "@/components/Layout";
 import Banner from "@/components/Banner";
 import { Config } from "../services/metadata/home";
 import MediaLayout from "@/components/MediaLayout";
+import { useEffect, useState } from "react";
+import { REACT_PLAYLIST_ID } from "config";
 
 export default function Home() {
-  const reactPlayLive = [
-    { id: 0, src: "https://www.youtube.com/embed/1qfDkmtuWqg" },
-    { id: 1, src: "https://www.youtube.com/embed/b0eas9xxD-E" },
-    { id: 2, src: "https://www.youtube.com/embed/w0nd4ASTDdg" },
-  ];
+  const [reactPlayLive, setReactPlayLive] = useState([]);
+
+  useEffect(() => {
+    const getPlayListData = async () => {
+      const response = await fetch(
+        `https://www.googleapis.com/youtube/v3/playlistItems?key=${process.env.NEXT_PUBLIC_YOUTUBE_API_KEY}&part=id,snippet&maxResults=5&playlistId=${REACT_PLAYLIST_ID}`
```

Comment:
Let's show Top 3.

Reply:
Made the changes. Now we are only fetching 3 latest videos from the playlist